contributing/development/compiling/compiling_with_mono.rst

```diff
@@ -172,197 +121,28 @@ The data directory is a dependency for Godot binaries built with the Mono module
 enabled. It contains important files for the correct functioning of Godot. It
 must be distributed together with the Godot executable.
 
-.. note:: The information below doesn't apply for Android, iOS and WASM,
-          as there is no data directory for these platforms.
-
-Export templates
```

Comment:
Does this information about export templates in MacOS no longer apply?

Reply:
The data directory for the export templates no longer exists at all and the bit about putting the editor specific `GodotSharp` directory into the .app is mentioned at the end of the editor section (line 132).